taxi/tests/test_forms.py

```diff
@@ -0,0 +1,122 @@
+from django.test import TestCase
+from django.contrib.auth import get_user_model
+from taxi.models import Driver, Manufacturer
+from taxi.forms import (
+    CarForm,
+    DriverCreationForm,
+    DriverLicenseUpdateForm,
+    DriverSearchForm,
+    CarSearchForm,
+    ManufacturerSearchForm,
+)
```

Comment:
Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability. 
Read more about topic [here](https://peps.python.org/pep-0008/#imports) 